demo-app/src/main/java/cash/z/ecc/android/sdk/demoapp/ui/screen/home/viewmodel/WalletViewModel.kt

```diff
@@ -226,8 +229,15 @@ class WalletViewModel(application: Application) : AndroidViewModel(application)
         if (null != synchronizer) {
             viewModelScope.launch {
                 val spendingKey = spendingKey.filterNotNull().first()
-                kotlin.runCatching { synchronizer.shieldFunds(spendingKey) }
-                    .onSuccess { mutableSendState.value = SendState.Sent(it) }
+                kotlin.runCatching {
+                    synchronizer.proposeShielding(spendingKey.account, Zatoshi(100000))?.let {
```

Comment:
Magic constant duplicated in more than one place. (I know this is just the demo app.)